smpte.js

```diff
@@ -632,44 +637,80 @@ function insertConformance(docMetadata) {
+
+  <p>The keywords, "should" and "should not" indicate that, among several possibilities, one is recommended
+    as particularly suitable, without mentioning or excluding others; or that a certain course of action
+    is preferred but not necessarily required; or that (in the negative form) a certain possibility
+    or course of action is deprecated but not prohibited.</p>
+
+  <p>The keywords "may" and "need not" indicate courses of action permissible within the limits of the document. </p>
+
+  <p>The keyword "reserved" indicates a provision that is not defined at this time, shall not be used,
+    and may be defined in the future. The keyword "forbidden" indicates "reserved" and in addition
+    indicates that the provision will never be defined in the future.</p>
 
-  <p>All text in this document is, by default, normative, except: the Introduction, any clause explicitly
-  labeled as "Informative" or individual paragraphs that start with "Note:" </p>
+  ${implConformance}
 
-<p>The keywords "shall" and "shall not" indicate requirements strictly to be followed in order to conform to the
-document and from which no deviation is permitted.</p>
+  <p>Unless otherwise specified, the order of precedence of the types of normative information in
+    this document shall be as follows: Normative prose shall be the authoritative definition;
+    tables shall be next; then formal languages; then figures; and then any other language forms.</p>
+    `;
 
-<p>The keywords, "should" and "should not" indicate that, among several possibilities, one is recommended
-  as particularly suitable, without mentioning or excluding others; or that a certain course of action
-  is preferred but not necessarily required; or that (in the negative form) a certain possibility
-   or course of action is deprecated but not prohibited.</p>
+  } else {
+
+    sec.innerHTML = `
+    <h2>Conformance</h2>
+    <p>This Engineering Guideline is purely informative and meant to provide tutorial information to the 
+    industry. It does not impose Conformance Requirements and avoids the use of Conformance Notation.</p>
 
-<p>The keywords "may" and "need not" indicate courses of action permissible within the limits of the document. </p>
+    <p>Engineering Guidelines frequently provide tutorial information about a Standard or Recommended Practice
+    and when this is the case, the user should rely on the Standards and Recommended Practices referenced for
+    interoperability information.</p>
+    `;
 
-<p>The keyword "reserved" indicates a provision that is not defined at this time, shall not be used,
-  and may be defined in the future. The keyword "forbidden" indicates "reserved" and in addition
-   indicates that the provision will never be defined in the future.</p>
+  }
 
-${implConformance}
+  if (docMetadata.pubType === smpte.EG_PUBTYPE) {
+    
+    const sections = document.querySelectorAll('section');
+    sections.forEach((element) => {
```

Comment:
use `for...of` when the body of the loop is huge.

Reply:
I tired this the first time around, but the element links don't pass through via the `logger_.error` messages.

Reply:
This is updated in https://github.com/SMPTE/html-pub/pull/287/commits/fc85e0020123c55a73bb7983c505c3bd0164ff77